include/aspect/material_model/interface.h

```diff
@@ -840,7 +840,7 @@ namespace aspect
          * 0, then the viscosity does not need to be computed.
          */
         virtual void evaluate(const MaterialModel::MaterialModelInputs<dim> &in,
-                              MaterialModel::MaterialModelOutputs<dim> &out) const = 0;
+                              MaterialModel::MaterialModelOutputs<dim> &out) const;
```

Comment:
Thank you Rene for looking into it. I modified the evaluate function to the older version. I am sorry but I don't know how to address the second issue. Could you help me into it?